snmp/metadata.csv

```diff
@@ -632,14 +632,15 @@ snmp.ltmNodeAddrMonitorState,gauge,,,,[F5 BIG-IP] The internal TMM monitor state
 snmp.ltmNodeAddrMonitorStatus,gauge,,,,[F5 BIG-IP] The load balancing state for a given node (including parent status).,0,snmp,,
 snmp.ltmNodeAddrNumber,gauge,,,,[F5 BIG-IP] The current number of nodes.,0,snmp,,
 snmp.ltmNodeAddrRatio,gauge,,,,[F5 BIG-IP] The load balancing ration on a given node.,0,snmp,,
+snmp.ltmNodeAddrSessionStatus,gauge,,,,[F5 BIG-IP] TODO,0,snmp,,
```

Comment:
Could you add a description to this metric ? Example:

https://www.circitor.fr/Mibs/Html/F/F5-BIGIP-LOCAL-MIB.php#ltmNodeAddrSessionStatus